src/Components/Results/Comparison.js

```diff
@@ -0,0 +1,63 @@
+const Comparison = ({ userWeather, uncleWeather, weatherSelection }) => {
+  const {
+    temperatureSelected,
+    cloudSelected,
+    windSelected,
+    humiditySelected,
+    twilightSelected
+  } = weatherSelection
+
+  const comparison = calcDiff(userWeather, uncleWeather);
+
+  function calcDiff(weatherObj1, weatherObj2) {
+    const diffObj = {};
+    for (let prop in weatherObj2) {
```

Comment:
I'm not sure about this but I might not use the word prop here as your variable (so as not to confuse it with the other props)